src/main/perl/Kerberos.pm

```diff
@@ -314,6 +319,18 @@ sub get_context
         return if(! defined($name));
     }
 
+    my $cred;
+
+    if ($opts{usecred}) {
+        $self->verbose("trying to get credential");
+        $cred = $self->get_cred(name => $name);
+        # Already reports error
+        return if ! $cred;
+    } else {
+        $self->verbose("No checking for credential");
```

Comment:
This adds a message to every invocation if not using the new option. Should this be debug or is it important enough to be a warning? Also, "No" -> "Not" ?